examples/hra_dreambooth/utils/args_loader.py

```diff
@@ -0,0 +1,363 @@
+import argparse
```

Comment:
The new utils files are basically copied from the existing ones, right? Let's add a comment to the files: `# copied from ...`  or `# adapted from ...`.

Reply:
Done.